src/app/onto-search/search/search.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="search-wrapper" fxLayout="row" fxLayoutAlign="space-between center">
```

Comment:
So, we tied ourselves to the flex-layout as well. What's the benefit in using this library here in this project?